packages/language-server/src/plugins/typescript/service.ts

```diff
@@ -148,13 +149,19 @@ async function createLanguageService(
         readDirectory: svelteModuleLoader.readDirectory,
         getDirectories: ts.sys.getDirectories,
         useCaseSensitiveFileNames: () => ts.sys.useCaseSensitiveFileNames,
-        getScriptKind: (fileName: string) => getSnapshot(fileName).scriptKind
+        getScriptKind: (fileName: string) => getSnapshot(fileName).scriptKind,
+        getProjectVersion: () => projectVersion.toString()
     };
+
     let languageService = ts.createLanguageService(host);
     const transformationConfig = {
         transformOnTemplateError: docContext.transformOnTemplateError
     };
 
+    docContext.globalSnapshotsManager.onChange(() => {
```

Comment:
Is this enough? When does a project version need to change? Looking at the code in the global snapshot manager, it doesn't emit a onChange event when someone updates a TS or JS file in `updateTsOrJsFile`.

Reply:
I think I'll just add the emit to the `updateTsOrJsFile` if branch that doesn't call the set method. Or do you prefer I add another event?

The project version needs to be updated every time there's a file change. So that typescript would rebuild.

Reply:
Adding the emit event to the branch where it's not called yet sounds good, don't think another event is needed.